### DISCUSSION TITLE
v2.4.2

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1145] [Announcements]
## What's Changed
* fix(ui): show changed status for toggling autoformatting and tabline by @mehalter in https://github.com/AstroNvim/AstroNvim/pull/1144

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v2.4.1...v2.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v2.4.2'>v2.4.2</a>.</em>

**Comment by karastoyanov** [comment]
Hello,
I am trying to update AstroNvim, running `:AstroUpdate`, but I am getting an error, something with `atempt to concatenate local 'tag' (a nill value)`. The issue comes from `/lua/core/utils/git.lua:88`
Any hints? I am running Windows 11 on neovim 0.8.0. Current version of AstroNvim is 2.0.2

**Reply by mehalter** [reply]
@karastoyanov This was an issue with windows stuff and the updater (Issue #1052) This has been fixed, but will require you to go and update the folder manually to get the new version with `git pull`. Sorry for the issue! The updater for windows should be working correctly now :)

**Reply by karastoyanov** [reply]
Yep, that did the job. After I've `pull`-ed the last version everything is working.

Thank you! Keep going!

**Comment by skshahriarahmedraka** [comment]
How to add Autosave plugin in Astrovim?


I am new to the vim/neovim community. I wanted to add Autosave feature like Vscode. I don't know the lua language. I broke Astrovim config file many times. and had to reinstall altogether.
I found this package but was unable to find any reliable solution that how to add this plugin in Astrovim.
https://github.com/Pocco81/AutoSave.nvim

**Reply by mehalter** [reply]
@skshahriarahmedraka I would recommend taking a look at the documentation for setting up a user configuration: https://astronvim.github.io/Configuration/manage_user_config

Then here is where you would add plugins in your `user/init.lua` file: https://github.com/AstroNvim/AstroNvim/blob/5288bad46858f15f2290aeb3d09c8bd8e58b7882/lua/user_example/init.lua#L216-L240

**Reply by skshahriarahmedraka** [reply]
thanks a lot
